rust/tests/integration/plain_transport.rs

```diff
@@ -403,6 +412,105 @@ fn create_non_bindable_ip() {
+                    send_buffer_size: None,
+                    recv_buffer_size: None,
+                }))
+                .await,
+            Err(RequestError::Response { .. }),
+        ));
+    });
+}
+
+#[cfg(not(target_os = "windows"))]
+#[test]
+fn create_two_transports_binding_to_same_ip_port_with_udp_reuse_port_flag_succeed() {
+    future::block_on(async move {
+        let (_worker, router) = init().await;
+
+        let multicast_ip = "224.0.0.1".parse().unwrap();
+        let port = pick_unused_port().unwrap();
+
+        // Transport 1.
+        let _ = router
+            .create_plain_transport({
+                PlainTransportOptions::new(ListenInfo {
+                    protocol: Protocol::Udp,
+                    ip: multicast_ip,
+                    announced_ip: None,
+                    port: Some(port),
+                    flags: Some(SocketFlags {
+                        ipv6_only: false,
+                        udp_reuse_port: true,
+                    }),
+                    send_buffer_size: None,
+                    recv_buffer_size: None,
+                })
+            })
+            .await
+            .expect("Failed to create first Plain transport");
+
+        // Transport 2.
+        let _ = router
+            .create_plain_transport({
+                PlainTransportOptions::new(ListenInfo {
+                    protocol: Protocol::Udp,
+                    ip: multicast_ip,
+                    announced_ip: None,
+                    port: Some(port),
+                    flags: Some(SocketFlags {
+                        ipv6_only: false,
+                        udp_reuse_port: true,
+                    }),
+                    send_buffer_size: None,
+                    recv_buffer_size: None,
+                })
+            })
+            .await
+            .expect("Failed to create second Plain transport");
+    });
+}
+
+#[cfg(not(target_os = "windows"))]
+#[test]
+fn create_two_transports_binding_to_same_ip_port_without_udp_reuse_port_flag_fails() {
+    future::block_on(async move {
+        let (_worker, router) = init().await;
+
+        let multicast_ip = "224.0.0.1".parse().unwrap();
+        let port = pick_unused_port().unwrap();
+
+        // Transport 1.
+        let _ = router
+            .create_plain_transport({
+                PlainTransportOptions::new(ListenInfo {
+                    protocol: Protocol::Udp,
+                    ip: multicast_ip,
+                    announced_ip: None,
+                    port: Some(port),
+                    flags: Some(SocketFlags {
+                        ipv6_only: false,
+                        udp_reuse_port: false,
+                    }),
+                    send_buffer_size: None,
+                    recv_buffer_size: None,
+                })
+            })
+            .await
+            .expect("Failed to create first Plain transport");
+
+        // Transport 2.
+        assert!(matches!(
```

Comment:
And this test fails because, as explained above, when `flags` is passed (other than `flags: None`) it doesn't matter whether given `udp_reuse_port` is true or false, because the worker always receives `true`.

Reply:
NOTE: As explained above this is not true. Flag value is properly passed to C++. Problem is that the test doesn't fail (it DOES fail in Node as expected).

Reply:
Fixed here: https://github.com/versatica/mediasoup/pull/1291/commits/7b935b61136793c0f1644ad3fcef27b65234d9c3